slurm/slurm_template.sh

```diff
@@ -1,41 +1,17 @@
 #!/bin/bash --login
 
-#SBATCH --time=3-00:00:00   # walltime
-#SBATCH --ntasks-per-node=8 # number of processor cores (i.e. tasks)
+#SBATCH --time=10:00:00   # walltime
+#SBATCH --ntasks-per-node=1# number of processor cores (i.e. tasks)
 #SBATCH --nodes=1   # number of nodes
-#SBATCH --mem=512G   # memory per CPU core
-#SBATCH --gres=gpu:8
-#SBATCH --qos=dw87
-#SBATCH -J "template_name"   # job name
+#SBATCH --gres=gpu:1
+#SBATCH --qos=cs
+#SBATCH --mem=64G   # memory per CPU core
+#SBATCH -J "train"   # job name
 #SBATCH --output=%x_%j.out
 
 # Set the max number of threads to use for programs using OpenMP. Should be <= ppn. Does nothing if the program doesn't use OpenMP.
 export OMP_NUM_THREADS=$SLURM_CPUS_ON_NODE
```

Comment:
Not necessarily, we likely weren't using OpenMP anyways for anything, QOS wouldn't change that. This is here simply because ORC recommends it, but we could remove it.

Reply:
Update on this: removing this line can result in strange behaviour.